src/Compilers/VisualBasic/Portable/Symbols/BaseTypeAnalysis.vb

```diff
@@ -190,7 +190,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
             Debug.Assert(root.OriginalDefinition = root, "root must not be a substitution")
 
-            If current Is Nothing Then
+            If current Is Nothing Or TypeOf current Is MissingMetadataTypeSymbol Then
```

Comment:
It feels like we shouldn't be getting this far with a `MissingMetadataTypeSymbol`